src/Calculator/Views/DateCalculator.xaml.cpp

```diff
@@ -186,7 +186,21 @@ void DateCalculator::SetDefaultFocus()
 void DateCalculator::DateCalcOption_Changed(_In_ Platform::Object ^ sender, _In_ Windows::UI::Xaml::Controls::SelectionChangedEventArgs ^ e)
 {
     FindName("AddSubtractDateGrid");
-    DateCalculationOption->SelectionChanged -= m_dateCalcOptionChangedEventToken;
+    auto dateCalcViewModel = safe_cast<DateCalculatorViewModel ^>(this->DataContext);
+
+    // https://github.com/microsoft/calculator/issues/254
```

Comment:
nit: you don't have to add this link when you work on new features

Reply:
Gotcha, thanks.